examples/pxScene2d/src/Makefile

```diff
@@ -264,6 +268,7 @@ PXSCENE_LIB_SRCS=\
   pxImageA.cpp \
   pxArchive.cpp \
   pxFont.cpp \
+  pxServiceManager.cpp \
```

Comment:
pxServiceManager.cpp should be conditionally added based on the SERVICE_MANAGER environment variable.  Please refer to the comment above.